ui/src/app/sensors/components/utils.ts

```diff
@@ -0,0 +1,28 @@
+import {Sensor} from '../../../models';
+
+export const Utils = {
+    statusIconClasses(sensor: Sensor): string {
```

Comment:
can we use this same colouring as on the events page?

Reply:
I feel like events page coloring needs to be changed.

Right now sensor list page uses the same coloring as workflows:
```
Init/Running -> Grey
Successful -> Green
Failed -> Red
```

Events page:
```
Init/Running -> Grey
Successful -> Blue
Failed -> Yellow
```

Thoughts?

```

Reply:
Running is blue workflows. 

But more generally, we use color to bring attention to things. Red is the most important color - as it tells you something needs sorting out. Other colors are much less important.  How about:

Init -> Grey
Running -> Blue
Successful -> Green
Failed -> Red

Reply:
The status of Sensor is reconciliation status, it doesn't represent the status of sensor application. So there's no real `Running` status (because reconciliation process is very fast), just `not ready`, which could be the same as `init`. @alexec

Reply:
we only want to show color if it is meaningful and actionable - what might that be?

Reply:
`Ready (green)` - Sensor is good.
`NotReady (grey)` - This tells the reconciliation didn't happen, might be something wrong with controller.
`Failed (red)` - There's something wrong with the Sensor spec.